### PR TITLE
Use bless v4.0

### PR DESCRIPTION
**Pull request by aabenoja**
The current version (v3.0.3) takes a significant amount of time on very large css files in terms of file size. His 4.0 branch is faster. I'm having bless pointed at my fork's dev branch instead since there are a few changes that were needed in order for 4.0 to work with our css.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   ],
   "dependencies": {
     "ansi-color": "~0.2.1",
-    "bless": "^3.0.3",
+    "bless": "git://github.com/aabenoja/bless.js.git#4.0.0-development",
```
Comment (mtscout6) [inline package.json]: What is available in your fork that is not in the published npm? Is there an associated pull request for that change that should be linked here?

Comment (aabenoja) [inline package.json]: I didn't want to publish the author's work on v4. He's also been very sporadic with his work on the update.

All his work has also been on a Dev branch for the past year

**Comment by mtscout6** [comment]
@aabenoja Could you follow up with @paulyoung and at least submit a pull request which we can track from here. I'm uncomfortable with taking on a git repo in dependency version. This is a great feature of npm when developing/debugging a third party dependency, however not to be encouraged when pushing to npm. I would at least prefer to see that some collaboration attempt has occurred with the dependency owner before pushing this.

**Comment by aabenoja** [comment]
PR sent to paulyoung/bless.js#44.

**Comment by aabenoja** [comment]
Maybe in the meantime I could create a github release containing my changes and change the target to that tarball? Just something for the interim until we get a response from Paul.

**Comment by paulyoung** [comment]
Hey guys.

Sorry if the hold up is causing problems for you. Unfortunately I can't prioritize work on bless right now.

I have a one or two more things that I want to add to v4 before it would be in a good place to accept contributions and maybe release a beta.

**Comment by mtscout6** [comment]
@aabenoja I like your idea of pointing at an actual release, that feels more reliable than just getting the tarball for a branch that can be changing. Go ahead and put that together until @paulyoung is able to get his beta out.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   ],
   "dependencies": {
     "ansi-color": "~0.2.1",
-    "bless": "^3.0.3",
+    "bless": "https://github.com/aabenoja/bless.js/archive/v4.0.0-alpha.tar.gz",
```
Comment (aabenoja) [inline package.json:29]: Created the github pre-release [here](https://github.com/aabenoja/bless.js/archive/v4.0.0-alpha.tar.gz). I'll change this to a semver value once the (official) beta gets pushed up.

**Pull request merged**